vhdl_lang/src/ast/search.rs

```diff
@@ -1075,14 +1075,27 @@ impl Search for Declaration {
                 return_if_found!(open_info.search(ctx, searcher));
                 return_if_found!(file_name.search(ctx, searcher));
             }
-
             Declaration::Package(ref package_instance) => {
                 return_if_found!(package_instance.search(ctx, searcher));
             }
-
+            Declaration::PackageDeclaration(ref decl) => {
+                return_if_found!(decl.search(ctx, searcher));
+            }
+            Declaration::PackageBody(ref body) => {
+                return_if_found!(body.search(ctx, searcher));
+            }
             Declaration::Configuration(_) => {
                 // @TODO
             }
+            Declaration::Disconnection(disconnect) => {
+                let DisconnectionSpecification {
+                    ident: _,
```

Comment:
You could now also search the `ident` here

Reply:
Done. Searching the references of the ident(s) is seems to be working now.